src/components/partnership/dto/update-partnership.dto.ts

```diff
@@ -0,0 +1,46 @@
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { DateTime } from 'luxon';
+import { Field, ID, InputType, ObjectType } from 'type-graphql';
+import { DateTimeField } from '../../../common';
+import { PartnershipAgreementStatus } from './partnership-agreement-status.enum';
+import { PartnershipType } from './partnership-type.enum';
+import { Partnership } from './partnership.dto';
+
+@InputType()
+export abstract class UpdatePartnership {
+  @Field(() => ID)
+  readonly id: string;
+
+  @Field(() => PartnershipAgreementStatus, { nullable: true })
+  readonly agreementStatus?: PartnershipAgreementStatus;
+
+  @Field(() => PartnershipAgreementStatus, { nullable: true })
+  readonly mouStatus?: PartnershipAgreementStatus;
+
+  @DateTimeField({ nullable: true })
+  readonly mouStart?: DateTime;
+
+  @DateTimeField({ nullable: true })
+  readonly mouEnd?: DateTime;
+
+  @Field(() => ID, { nullable: true })
+  readonly organizationId?: string;
```

Comment:
```suggestion
```
Partnership's Organization cannot be changed. That's more like an entirely new partnership.